packages/types/src/ts-estree.ts

```diff
@@ -464,20 +462,8 @@ export type PrimaryExpression =
   | TemplateLiteral
   | ThisExpression
   | TSNullKeyword;
-export type ProgramStatement =
-  | ClassDeclaration
-  | ExportAllDeclaration
-  | ExportDefaultDeclaration
-  | ExportNamedDeclaration
-  | ImportDeclaration
-  | Statement
-  | TSDeclareFunction
-  | TSEnumDeclaration
-  | TSExportAssignment
-  | TSImportEqualsDeclaration
-  | TSInterfaceDeclaration
-  | TSNamespaceExportDeclaration
-  | TSTypeAliasDeclaration;
+/** @deprecated TODO: delete this in next major release */
```

Comment:
in the next major we should be seeking to re-align this with EStree spec, not deleting this type.

Reply:
updated